.changeset/cool-cherries-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@contentful/f36-header': major
```

Comment:
This will get the version on package.json and bump a major version, which will cause it to go to 5.0.0, and trigger it for all of the fixed packages

Reply:
You can test how changeset will update the version by running `npx changeset version`, just restore the files after running it 😅